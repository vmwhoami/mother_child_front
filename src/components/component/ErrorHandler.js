import React from 'react';
import PropTypes from 'prop-types';
import css from '../../css/errorsSuccess.module.css';

const ErrorHandler = ({ errors }) => (
  <div className={css.erorrs}>
    {
      errors.map((error) => <span key={`${error}_1`} className={css.error}>{error}</span>)
    }
  </div>
);

ErrorHandler.propTypes = {
  errors: PropTypes.instanceOf(Array).isRequired,
};
export default ErrorHandler;
