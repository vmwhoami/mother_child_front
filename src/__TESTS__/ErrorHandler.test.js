import React from 'react';
import renderer from 'react-test-renderer';
import ErrorHandler from '../components/comp/ErrorHandler';
const errors = ["First error", "Second error"]

it('renders Error correctly', () => {
  const tree = renderer.create(<ErrorHandler errors={errors} />);
  expect(tree).toMatchSnapshot();
});