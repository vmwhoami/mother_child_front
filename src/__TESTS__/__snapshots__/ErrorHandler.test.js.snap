// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Error correctly 1`] = `
<div
  className="erorrs"
>
  <span
    className="error"
  >
    First error
  </span>
  <span
    className="error"
  >
    Second error
  </span>
</div>
`;
